components/econet/econet.cpp

```diff
@@ -424,8 +431,16 @@ void Econet::transmit_message_(uint8_t command, const std::vector<uint8_t> &data
   tx_message_.push_back(crc);
   tx_message_.push_back(crc >> 8);
 
+  if (this->flow_control_pin_ != nullptr) {
+    this->flow_control_pin_->digital_write(true);
+  }
+
   this->write_array(&tx_message_[0], tx_message_.size());
-  // this->flush();
```

Comment:
Uncommenting this seems to change behavior even when a flow control pin is not added; is that intentional?

Reply:
I could move it in the condition below but it's better to flush for both cases. It's good to immediately flush to ensure all bytes are transmitted together and avoid collisions with other messages, incoming or outgoing.